docs/content/flink/procedures.md

```diff
@@ -488,20 +488,25 @@ All available procedures are listed below.
       <td>
          -- for Flink 1.18<br/>
          -- clear table with purge files directly.<br/>
-         CALL [catalog.]sys.purge_files('identifier')<br/><br/>
+         CALL [catalog.]sys.purge_files('identifier')<br/>
+         -- only check what dirs will be deleted, but not really delete them.<br/>
+         CALL [catalog.]sys.purge_files('identifier', true)<br/><br/>
          -- for Flink 1.19 and later<br/>
          -- clear table with purge files directly.<br/>
-         CALL [catalog.]sys.purge_files(`table` => 'default.T')<br/><br/>
+         CALL [catalog.]sys.purge_files(`table` => 'default.T', `dry_run` => true)<br/><br/>
```

Comment:
Just noticed that the description and the example code does not match. In order to "clear table with purge files directly", dry_run should be false.